cluster/src/main/scala/com/linkedin/norbert/cluster/Node.scala

```diff
@@ -89,7 +95,7 @@ final case class Node(id: Int, url: String, available: Boolean, partitionIds: Se
     case _ => false
   }
 
-  override def toString = "Node(%d,%s,[%s],%b,0x%08X,0x%08X)".format(id, url, partitionIds.mkString(","), available, if (capability.isEmpty) 0L else capability.get, if (persistentCapability.isEmpty) 0L else persistentCapability.get)
+  override def toString = "Node(%d,%s,[%s],%b,0x%08X,0x%08X,%d)".format(id, url, partitionIds.mkString(","), available, if (capability.isEmpty) 0L else capability.get, if (persistentCapability.isEmpty) 0L else persistentCapability.get, if (offset.isEmpty) 0 else offset.get)
```

Comment:
Yup, I'll replace 0 here with -1. I'm checking if the offset parameter is set during protobuf decoding, so this isn't an issue there.